src/js/components/search/visualizations/rank/RankVisualizationSection.jsx

```diff
@@ -73,15 +73,15 @@ export default class RankVisualizationSection extends React.Component {
         let disablePrev = false;
         let hidePager = '';
 
-        if (this.props.total < this.props.page + 1) {
+        if (!this.props.hasNextPage) {
```

Comment:
Since this check is now on the truthiness of a boolean, we can just set `disableNext = !this.props.hasNextPage;` and `disablePrev = !this.props.hasPreviousPage;` above